openapi_tester/loaders.py

```diff
@@ -258,3 +259,28 @@ def load_schema(self) -> dict[str, Any]:
             return cast(
                 "dict", json.loads(content) if ".json" in self.path else yaml.load(content, Loader=yaml.FullLoader)
             )
+
+
+class UrlStaticSchemaLoader(BaseSchemaLoader):
+    """
+    Loads OpenAPI schema from an url static file.
+    """
+
+    def __init__(self, url: str, field_key_map: dict[str, str] | None = None):
+        super().__init__(field_key_map=field_key_map)
+        self.url = url
+
+    def load_schema(self) -> dict[str, Any]:
+        """
+        Loads a static OpenAPI schema from url, and parses it to a python dict.
+
+        :return: Schema contents as a dict
+        :raises: ImproperlyConfigured
+        """
+        response = httpx.get(self.url)
```

Comment:
Better to use a builtin library like requests

Reply:
Replaced `httpx` with `requests`